examples/impute/plot_missing_values.py

```diff
@@ -31,16 +32,29 @@
 
 rng = np.random.RandomState(0)
 
+CV_SPLIT_NUM = 5
```

Comment:
just call this N_SPLITS (consistent with the terminology in `sklearn.model_selection._split`)

Reply:
Will do.